src/main/resources/templates/registration.html

```diff
@@ -1,163 +1,190 @@
+                                <label class="user__label">
+                                    <input class="user__input" type="email" name="email" placeholder="Email" required>
+                                </label>
+                                <label class="user__label">
+                                    <input class="user__input" type="password" name="password" placeholder="Password"
+                                           required id="password" onblur="checkRegex(this)">
+                                </label>
+                                <script language='javascript'>
+                                    function checkRegex(passwordInput) {
+                                    var varRegex = new RegExp("(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}");
+                                        if(passwordInput.value.match(varRegex))
+                                        {
+                                            document.getElementById('user_message').innerHTML=" ";
+                                            return true;
+                                        }
+                                        else
+                                        {
+                                            document.getElementById('user_message').innerHTML="Password must be at least 8 characters long and contain numbers, lowercase and uppercase letters!";
+                                            document.form_reg.password.focus();
+                                            return false;
+                                        }
+                                    }
+                                </script>
+                                <label class="user__label">
+                                    <input class="user__input" type="password" name="confirmPassword" id="confirmPassword"
+                                           placeholder="Confirm password" required>
+                                </label>
+                                <script language='javascript'>
+                                    function matchPassword() {
+                                        if(document.form_reg.password.value != document.form_reg.confirmPassword.value) {
+                                            document.getElementById('user_message').innerHTML="Sorry, the passwords don't match!";
+                                            document.form_reg.confirmPassword.focus();
+                                            return false;
+                                        }
+                                        document.form_reg.submit();
+                                    }
+                                </script>
+                                <label class="user__label">
+                                    <input class="user__input" type="text" name="discogsUserName"
+                                           placeholder="Discogs User Name">
```

Comment:
"When referring to an identity used with a password, "username" is used. If you are referring to a real name or full name, "user name" could be used. As far as "user-name," it is rarely used. In our earlier example, "John Smith" is the user name, and "smitj" is the username used with his password." (https://www.computerhope.com/jargon/u/username.htm)

Username, not User Name